src/pages/LegacyIntegrationNotification.vue

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+import Vue from 'vue';
+import { ipcRenderer } from 'electron';
+import paths from '@/utils/paths';
+
+export default Vue.extend({
+  layout:     'dialog',
+  computed: {
+    oldIntegrationPath() {
+      return paths.oldIntegration;
+    },
+  },
+  mounted() {
+    ipcRenderer.send('dialog/ready');
+  },
+  methods: {
+    close() {
+      window.close();
+    }
+  },
+});
+</script>
+
+<template>
+  <div>
+    <h3>{{ t('legacyIntegrations.title') }}</h3>
```

Comment:
I think you're on the right track with providing more information and a `v-tooltip` in the header might make sense to help answer some questions. With the current messaging, I might be a bit confused as to what I will actually need to remove.

Reply:
Ok I added a tooltip. Let me know what you think!